registry/handlers/app.go

```diff
@@ -577,3 +578,7 @@ func startUploadPurger(storageDriver storagedriver.StorageDriver, log ctxu.Logge
 	}()
 
 }
+
+func (app *App) logError(context context.Context, status int, content interface{}) {
```

Comment:
I thought of that; the problem is when there are multiple errors; we would lose some info, given that context keys are unique.

Unless we generated the keys: (err1.code, err2.code) etc, but that seemed overkill.  I'm OK either way though.